ietf/settings.py

```diff
@@ -947,6 +947,7 @@ def skip_unreadable_post(record):
 FLOORPLAN_DIR = os.path.join(MEDIA_ROOT, FLOORPLAN_MEDIA_DIR)
 
 MEETING_USES_CODIMD_DATE = datetime.date(2020,7,6)
+MEETING_LEGACY_OFFICE_HOURS_END = 111  # last meeting to use legacy office hours representation
```

Comment:
1:11 is a very odd time for the office hours to end...

Reply:
True